CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 * Allow to change the state of word embeddings optimization during a retraining
 * Check consistency of option settings when training from existing model
 * Save and Restore random number generator states in checkpoints
+* fix BPE usage from REST server
```

Comment:
As the BPE changes are not yet released, there is no need to add this in the changelog.